projects/packages/blaze/src/class-blaze.php

```diff
@@ -61,22 +61,22 @@ public function register() {
 	 *
 	 * @return array
 	 */
-	public function jetpack_promote_posts_row_action( $post_actions, $post ) {
+	public function jetpack_blaze_row_action( $post_actions, $post ) {
 		$post_id = $post->ID;
 
 		// @todo wrap in method call, for general use?
 		$promote_post_url = Redirect::get_url(
-			'jetpack-promote-posts',
+			'jetpack-blaze',
 			array(
 				'query' => 'blazepress-widget=post-' . esc_attr( $post_id ),
 			)
 		);
 
 		// Add the link, make sure to tooltip hover.
-		$text  = _x( 'Promote', 'Verb', 'jetpack-promote-posts' );
+		$text  = _x( 'Promote', 'Verb', 'jetpack-blaze' );
```

Comment:
Should this and the following printed wording be 'promote', or 'blaze'?

Reply:
Will update the UI in a following PR, thanks!